### ISSUE TITLE
Phing plugin not working

### ISSUE THREAD

**Issue by williameggers-catch-e** [php-censor/php-censor#304]
## Expected behavior

Phing plugin to operate without error.

## Actual behavior

RUNNING PLUGIN: Phing (Stage: Success)
**Error: Call to undefined method PHPCensor\Plugin\Phing::getDirectory() (uncaught exception) at <PHP_CENSOR_PATH>/src/Plugin/Phing.php line 161**

Phing plugin appears to be calling a non-existent getDirectory() method that is missing/removed. 

Commenting out the following lines in propertiesToString() appears to resolve the issue...

```
        /*if (!isset($this->properties['project.basedir'])) {
            $this->properties['project.basedir'] = $this->getDirectory();
        }*/
```

## Steps to reproduce

Added the following to my .php-censor.yml file within my project and triggered a build.

```
success:
  phing:
    build_file: "build.xml"
```

## Environment

* PHP Censor version 1.0.9
* Operating System Amazon Linux 2018.08
* PHP version 7.1.28
* MySQL version 5.7.25

**Comment by corpsee** [comment]
@williameggers-catch-e Thank you for the report! I fixed plugin. Could you test your case on git branch `phing-directory-fix` please?

**Comment by williameggers-catch-e** [comment]
@corpsee Thanks for the fix. Switched branches and triggered a build. Works well.

I have another request for the Phing plugin, but it's an enhancement. I'll raise it separately.

**Comment by corpsee** [comment]
Changes merged into branch `release-1.0` and will be released with version `1.0.10` soon.
